### PR TITLE
Various UV/texture fixes for buttons

### LINKED ISSUE #528: [REQUEST] Implement the ANN LT
~~- [ ] make the test position require you to hold the mouse button down (its spring loaded, i think?)~~
- [x] dim and brt setting should change the brightness level of all the button that have lights
- [x] test position should make all button lights turn on #566

ANN LT TEST
- [x] make most buttons light up #566
- [x] make sure no double blue dots are visible on the buttons #566
- [x] make displays show test stuff
  - [x] FCU #590
    - [x] IRL has a decimal point to the left of every number, but the game FCU only has one's that are actually used, so we will have to add the other decimal elements to match IRL #590
  - [x] BARO display #590
  - [x] battery displays?
- [x] more button/square thingy lights
  - [x] ATC MSG x2
- [ ] make sure ANN LT TEST is in the checklist
- [x] sidestick priority arrows
- [ ] Overhead
  - [x] EVAC COMMAND (should have OFF for seq2, probably has blue dots in vanilla)
  - [x] ADIRS ON BAT (couldn't find anything in gltf or interior xml for it)
  - [x] ADIRS IR buttons should say OFF for seq2 (vanilla is ALIGN)
  - [x] PA should probably have something (vanilla just has blue dots for seq2 and nothing else)
  - [x] 5x FIRE AGENT buttons should say DISCH (vanilla says "neo")
  - [ ] 3x big rectangular FIRE things need emissive textures or something
  - [x] ELEC PUMP should have ON in white for seq2 (vanilla is blue ON)
  - [x] FUEL MODE SEL should have MAN in white (vanilla is MAN in blue)
  - [x] FUEL XFEED should have ON in white (vanilla is ON in blue)
  - [x] ELEC
    - [x] COMMERCIAL should not have anything in seq1 (vanilla has FAULT) (needs verification? maybe the light was broken in the vid?)
  - [x] AIR COND
    - [x] RAM AIR should have ON in white for seq2 (needs confirmation)
  - [x] CABIN PRESS MODE SEL should have MAN in white (in vanilla its blue)
  - [x] DITCHING has some kind of white light (vanilla is blank or blue dots)
  - [x] SIGNS has something in orange? but vanilla is a white OFF
- [x] Chronometer should have lots of 88888 #590
- [ ] MCDU has 5 lights on top and some on the sides
- [x] radio thing should show all 88888888 #590
- [x] push buttons by engine starters
- [x] COCKPIT DOOR square thing should not have green on it
- [x] RUD TRIM should show something like R 88.8
- [x] ATC FAIL light should be on #590
- [x] TCAS display should be 8888 #590
- [ ] down arrow by gear lever should light up red

# References
ATC MSG
image

FCU and other things: https://www.youtube.com/watch?v=RmuH2St76Rs

image

image

### LINKED ISSUE #956: [META] Emissive Textures
# Meta Issue for Emissive Texture Work

#### Engine Control Panel (Master Switches)
- [x] Both DUMMY buttons should say FIRE in RED & have FAULT in Amber. - *Ref. 2. Engine Control Panel Image*
#### CKPT DOOR
- [x] OPEN in Amber.
- [x] FAULT in RED.
#### GRAVITY GEAR
- [x] Text on turn handle shouldn't be lit up. - *Ref. 3. Image of Pedestal*
#### Transponder 
- [x] ATC FAIL should be AMBER not PINK. - *Ref. 4. Youtube video demo. https://www.youtube.com/watch?v=PyR6BXlYI2o*

## GLARESHIELD
- [x] Master Caution should be in Amber.
- [x] Side stick priority should have ARROW pointing to who has priority in RED - *Ref. 5. Image of priority*
- [x] Outermost button should have an "ATC MSG" emissive in blue - *Ref 1.*

## CLOCK
- [ ] RST, CHR, DATE, GPS/INT, RUN/STP/RST shouldn't light up. 

## SAI - *Ref. 7 Image of SAI*
- [x] ATT and BARO shouldn't light up. (After #1081 gets merged)

### CALLS
- [x] EMER should say CALL instead of OFF in AMBER on the top, and BOXED WHITE ON instead of OFF on the bottom

### OXYGEN
- [x] CREW SUPPLY should say "OFF" with TWO WHITE DOTS ABOVE.
- [x] PASSENGER should say "SYS ON" in WHITE with TWO DOTS BELOW.

### RCDR
- [x] GND CTL should have TWO BLUE DOTS above BOXED ON.

### GPWS
- [x] LDG FLAP 3 should say "ON" in WHITE with TWO WHITE DOTS ABOVE.
- [x] FLAP MODE should say "OFF" in WHITE with TWO WHITE DOTS ABOVE.
- [x] G/S MODE should say "OFF" in WHITE with TWO WHITE DOTS ABOVE.

### EMER ELEC PWR
- [x] RAT should be RED FAULT with TWO WHITE DOTS UNDER.
- [x] GEN 1 LINE SMOKE should be AMBER.

### EVAC
- [x] COMMAND should say "EVAC" in BOXED RED with BOXED ON UNDER.

### ADIRS 
- [x] ALIGN should say OFF in BOXED WHITE. *Ref. 9 Schematic of ADIRS*
- [x] ON BAT in AMBER.

### FIRE
- [x] DISCH should be BOXED AMBER.

### HYDRAULICS
- [x] ELEC PUMP should be BOXED WHITE ON.

### FUEL
- [x] MAN should be BOXED WHITE.
- [x] CTR PUMP 2 is missing FAULT light.
- [x] XFEED should be BOXED WHITE "ON".
- [ ] XFEED should be GREEN OPEN

### ELEC
- [x] COMMERCIAL should be BOXED OFF with TWO WHITE DOTS ABOVE.
- [x] IDG1 & 2 should have TWO WHITE DOTS UNDER.
- [x] BUS TIE should have TWO WHITE DOTS ABOVE.

### AIR COND.
- [x] RAM AIR should be BOXED ON with TWO WHITE DOTS ABOVE.
- [x] PROBE WINDOW should have TWO WHITE DOTS ABOVE.

### CABIN PRESS.
- [x] MODE SEL should be BOXED WHITE MAN.
- [x] DITCHING should be BOXED WHITE ON with TWO WHITE DOTS ABOVE.

### SIGNS
- [x] EMER EXIT LT should be AMBER OFF with TWO WHITE DOTS ABOVE.

### CARGO SMOKE
- [x] DISCH should be AMBER.

### VENTELATION
- [x] BLOWER should be BOXED "OVRD" in WHITE.
- [x] EXTRACT text missing and should be same as BLOWER.
- [x] CAB FANS should have WHITE DOTS ABOVE.

## OVERHEAD MAINTENANCE PANEL - *Ref. 8*
- [x] BLUE PUMP OVRD should be BOXED WHITE ON
- [x] Both FADEC GND PWR should be BOXED WHITE ON
- [x] APU AUTO EXT should be BOXED WHITE ON and WHITE OK
- [x] Both SVCE INT OVRD and AVIONICS COMPT LT should be BOXED WHITE ON
- [x] OXYGEN TMR RESET should be BOXED WHITE ON and WHITE FAULT

**References**

grafik
grafik
*Ref. 1 ATC MSG on glareshield*
image
*Ref. 2 Engine Control Panel (Below THROTTLE)*
image
*Ref. 3 Image of Pedestal showing GRAVITY GEAR isn't an EMISSIVE*
image
*Ref. 5 Image of SIDE STICK PRIORITY*
P1010761
*Ref. 6 Image of OVHD Panel (Zoom in for more detail, Source: komp)*
image
*Ref. 7 Image of SAI*
IMG_20201013_013135
*Ref. 8 Image of Overhead Maintenance Panel - Source: Party Grunt on discord*
grafik
*Ref. 9 Schematic of ADIRS*

## Closing Notes
If you need anymore references please reach out to me via this ISSUE or via DISCORD.

Discord username (if different from GitHub): Sabes#8668

### LINKED ISSUE #1410: [BUG] Meta issue for wrong textures and push buttons in the cockpit
### INTRO

**UPDATED 15-NOV-2020**

I took some time and compared the entire cockpit of the simulator with the real one. I'm using this issue to keep track of all the differences and inaccuracies of the Asobo model. I'm going to make a checkbox item for every issue I come across. Also feel free to remind me of anything that's missing.

I will take more and detailed pictures whenever possible. If you have high quality pictures and want to share them with me, hit me up.

**I _do know_ that our modelers and designers are not able to fix most of this for the time being. But it will come in handy once they are.**

Discord name: komp#1821

### REFERENCES
To start off with, I made this tiny PDF with comparison pictures of the overhead and center pedestal. Im going to continue it this way with the other facilities in the cockpit. This issue will receive regular updates.

[a32nx panel comparison.pdf](https://github.com/flybywiresim/a32nx/files/5393623/a32nx.panel.comparison.pdf)

_____________
_____________

### KNOWN ISSUES

**ADIRS panel**
- [ ] **_ADIRS panel:_** ON BAT annunciator light should light up during light test
- [x] **_ADIRS panel:_** IR pushbuttons don’t have ALIGN indication IRL
- [ ] **_ADIRS panel:_** ATT label of IR2 is covered by lightplate screw
_____________________
**EVAC&EMER ELEC PWR panel**
- [ ] **_EVAC&EMER ELEC PWR panel:_** EVAC COMMAND push button legends should be red EVAC and white ON
- [ ] **_EVAC&EMER ELEC PWR panel:_** (not visible) EMER GEN TEST pushbutton: below the cap should be the same type as the (round) call pushbuttons, not a Korry 
- [ ] **_EVAC&EMER ELEC PWR panel:_** (not visible) RAT MAN ON pushbutton: below the cap should be the same type as the (round) call pushbuttons, not a Korry
- [ ] **_EVAC&EMER ELEC PWR panel:_** SMOKE legend in GEN 1 LINE pushbutton should be amber, not red
- [ ] **_EVAC&EMER ELEC PWR panel:_** RAT & EMER GEN annunciator (not a pushbutton): FAULT legend should be red. Blue dots should not light up
_____________________
**GPWS panel**
- [ ] **_GPWS panel:_** LDG PLAP 3 pushbutton should have an ON legend rather than an OFF legend
_____________________
**OXYGEN & CALLS panel**
- [ ] **_OXYGEN & CALLS panel:_** Passenger oxygen system annunciator (not a pushbutton!) should be white
- [ ] **_OXYGEN & CALLS panel:_** (not visible) MASK MAN ON pushbutton below the cap should be the same type as the (round) call pushbuttons, not a Korry
- [ ] **_OXYGEN & CALLS panel:_** EMER call pushbutton should have an amber CALL legend rather than a blue ON
- [ ] **_OXYGEN & CALLS panel:_** WIPER label should be the same size as RAIN RPLNT
- [x] **_OXYGEN & CALLS panel:_** Position marks of the wiper switch scale should be bold
- **_OXYGEN & CALLS panel:_** _NOTE: the ALL CALL push button is optional. Most NEOs don’t have it_
_____________________
**PRESS & INT LT panel**
- [ ] **_PRESS & INT LT panel:_** CABIN PRESS title should be bigger 
- [ ] **_PRESS & INT LT panel:_** INT LT title should be bigger
- [ ] **_PRESS & INT LT panel:_** SIGNS title should be bigger
- [ ] **_PRESS & INT LT panel:_** MODE SEL pushbutton: MAN legend should be white
- [ ] **_PRESS & INT LT panel:_** Scale of LDG ELEV selector is messed up. Compare the pictures
- [ ] **_PRESS & INT LT panel:_** Ditching pushbutton should light up during light test 
- [x] **_PRESS & INT LT panel:_** Typo: COMPAS is missing an ‘S’
- [ ] **_PRESS & INT LT panel:_** ANN LT switch: TEST/DIM legends too far apart
- **_PRESS & INT LT panel:_** _NOTE: SEAT BELTS switch AUTO function is optional_ 
- [ ] **_PRESS & INT LT panel:_** EMER EXIT LT annunciator legend should be amber OFF and not boxed
_____________________
**AIR COND panel**
- [x] **_AIR COND panel:_** Position marks of the PACK FLOW selector scale should be bold
- [ ] **_AIR COND panel:_** RAM AIR pushbutton has no emissive texture during light test
- **_AIR COND panel:_** _NOTE: HEATER CAPT and HEATER F/O switches in the IRL picture are optional. If not installed, the lines left and right of the COCKPIT box are not present_
- [ ] **_AIR COND panel:_** There should be a small green line leading to the temperature selectors
_____________________
**ELEC panel**
- [ ] **_ELEC panel:_** COMMERCIAL pushbutton should not have a FAULT legend 
- [ ] **_ELEC panel:_**  Battery voltage displays are too wide
- [ ] **_ELEC panel:_**  AC ESS FEED pushbutton legend should be ALTN legend rather than OFF
_____________________
**HYD/FUEL panel**
- [ ] **_HYD/FUEL panel:_** CTR TK MODE SEL pushbutton “MAN” annunciator should be white
- [ ] **_HYD/FUEL panel:_** Battery voltage display is too wide
- [ ] **_HYD/FUEL panel:_** (not visible) RAT MAN ON pushbutton: below the cap should be the same type as the (round) call pushbuttons, not a Korry
_____________________
**CARGO SMOKE/VENT/ENG (ovhd) panel**
- [ ] **_CARGO SMOKE/VENT/ENG (ovhd) panel:_** (not visible) both DISCH pushbuttons: below the cap should be the same type as the (round) call pushbuttons, not a Korry
- [ ] **_CARGO SMOKE/VENT/ENG (ovhd) panel:_** Font size of CARGO SMOKE, VENTILATION, ENG, WIPER & RAIN RPLNT too small
- [ ] **_CARGO SMOKE/VENT/ENG (ovhd) panel:_** BLOWER pushbutton: OFF legend should be OVRD instead
- [ ] **_CARGO SMOKE/VENT/ENG (ovhd) panel:_** EXTRACT pushbutton: OFF legend should be OVRD instead
- [ ] **_CARGO SMOKE/VENT/ENG (ovhd) panel:_** ENG N1 MODE pushbuttons don’t exist in the NEO (LEAP & PW1100). These are exclusively for the CEO IAE V2500 engine
- [x] **_CARGO SMOKE/VENT/ENG (ovhd) panel:_** Position marks of the wiper switch scale should be bold

_____________________
**ENG (center pedestal) panel**
- [ ] **_ENG (center pedestal) panel:_** Missing label “MASTER 1 (2)” above the engine master switches
- [ ] **_ENG (center pedestal) panel:_** Engine annunciators (NO PUSH function!!) don’t have emissive texture during light test (red FIRE/amber FAULT)
- [x] **_ENG (center pedestal) panel:_** Position marks of the engine mode selector scale should be bold
_____________________
**Radio Management Panel**
- [ ] **_RMP:_** LOAD pushbutton is missing (used for CPDLC)
- [ ] **_RMP:_** STBY NAV label and line missing above the standby navigation pushbuttons
- [ ] **_RMP:_** Indicator lights should be above the pushbuttons and have the shape of a triangle
- [ ] **_RMP:_** SEL indicator should be amber
_____________________
**Audio Control Panel**
- [x] **_ACP:_** HF1, HF2 INT, CAB and PA microphone pushbuttons do not light up during light test or when selected
- [x] **_ACP:_** ON indicator in VOICE pushbutton does not light up during light test
- [ ] **_ACP:_** _NEW:_ VOICE push button should have an ON legend rather than three green lines
- [ ] **_ACP:_** Minor visual flaws with the rotary knobs
- **_ACP:_** Ignore the missing MLS label in the real picture. Let’s just pretend our Airbus is equipped with microwave landing system
_____________________
**ECAM Control Panel**
- [ ] **_ECP:_** All pushbutton indicator lights should be white

### PR DESCRIPTION
**Pull request by tyler58546**
Partially addresses #956, partially addresses #1410, partially addresses #528

## Summary of Changes

- Fixed DCDU button labels showing "MS0" instead of "MSG" (again)
- Fixed LDG FLAP 3 button showing OFF instead of ON
- Fixed CABIN PRESS MODE SEL button showing MAN in blue instead of white
- Fixed COMMERCIAL button showing FAULT instead of two dots
- Fixed buttons having blue dots instead of white dots
- Fixed YELLOW ELEC PUMP button showing ON in blue instead of white
- Fixed X FEED button showing ON in blue instead of white

## Screenshots (if necessary)

Before:
image

After:
image

## References

See #956 

## Testing instructions

Ensure that all changes listed above are present and do not cause any unintended texture issues.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by anthony61596 — APPROVED** [review]

**Comment by Benjozork** [comment]
> Partially fixes #956, partially fixes #1410, partially fixes #528

GitHub will still close those issues, can you reword it to say "Addresses" instead ?

**Comment by lukecologne** [comment]
The dots don't actually light up, see Ref. 6 from #956

**Comment by tyler58546** [comment]
> 
> 
> > Partially fixes #956, partially fixes #1410, partially fixes #528
> 
> GitHub will still close those issues, can you reword it to say "Addresses" instead ?

This has been addressed.

**Review by wpine215 — APPROVED** [review]

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: 10/1/21
Version of the sim: 1.12.13.0
PR Tested: #2965 
Object of test: Buttons

Testing technique: Loaded in and looked at all the buttons mentioned
Steps you've made to test the Pull Request:
1. Loaded in on the runway
2. Went through all the buttons listed above and checked they acted as described

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good
Conclusions of the testing: Everything works as described in the pr
